web/pgadmin/browser/server_groups/servers/databases/schemas/statistics/tests/test_statistics_delete.py

```
@@ -0,0 +1,159 @@
+
+            if self.is_list:
+                statistics_response_2 = statistics_utils.verify_statistics(
+                    self.server, self.db_name, self.statistics_name_2
+                )
+                self.assertIsNone(statistics_response_2,
+                                  "Second deleted statistics still present")
+        else:
+            if self.mocking_required:
+                with patch(
+                    self.mock_data["function_name"],
+                    side_effect=[eval(self.mock_data["return_value"])]
```

Ruff (0.15.15)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)